content/blog/Galaxy_Climate_Project_Introduction_Blog_1/index.md

```diff
@@ -0,0 +1,101 @@
+source_blog: "Galaxy Climate Outreachy project"
+
+date: "2022-07-04"
+---
+
+# Introduction to GALAXY CLIMATE 
+
+![Galaxy Climate Project Outreachy Blog1](Galaxy_Climate_Project_Outreachy_Blog1.png)
+
+Have you heard about the **Climate Science Workbench** of Galaxy? It is named **[GALAXY CLIMATE](https://climate.usegalaxy.eu/)**. This is the first blog in the series of five blog posts that I will be sharing. Let’s explore it in more detail.
+
+
+Aiming to make computational data analysis accessible to research scientists (from those who lack knowledge in computer programming or systems administration to the most advanced users), Galaxy is a platform for scientific workflow creation, data integration and data analysis. Despite being primarily domain-independent and used as a general workflow management system, it was initially created for genomics research. In recent times, Galaxy has initiated the inclusion and development of Climate-related tools under its toolshed.
+
+Galaxy Climate is a scientific workbench focusing on making climate analysis easier, in simpler words - “ **command-line free**”. It houses numerous tools, which makes the process a lot more convenient. The number of tools available for the purpose is rich and is increasing with demand. The supportive community which makes it possible is efficiently equipped with the required expertise and passion for the initiative.
+
+
+
+# Why is GALAXY focusing on CLIMATE SCIENCE?
+
+We live on Earth, but why? To state, it has the right temperature, air, and water conditions for our survival. It is an inevitable fact that climate is associated with our existence. And both go hand-in-hand. Technological advances can be of actual use if we can ensure our survival. The world is shifting to greener norms or a more sustainable tomorrow.
+
+Handling massive command-line-focused analysis of climate data can be exhaustive as well as cumbersome. Sometimes it may be system environments, and other times, it may be some technical jargon that has the probability of delaying the analysis. And not everyone is well equipped with coding skills. Nevertheless, a standard, easy-to-use, accurate, and scientifically sound platform is the need of the hour. The GALAXY CLIMATE workbench has almost met all the criteria for improving the situation. And it is still growing.
+
+
+
+
+# What are the existing CLIMATE TOOLS available in the GALAXY TOOLSHED?
+
+The Galaxy ToolShed houses more than 2,000 tools, out of which several tools are helpful for Climate Analysis.  Some of the relevant tags to find under the Galaxy ToolShed are: “Climate Analysis”, “Ecology”, etc. You can follow the list given below for more clarity. 
+
+
+## CLIMATE ANALYSIS TOOLS
+| Climate Analysis Tools |      Synopsis    |
+|----------|:-------------:|
+| [c3s](https://toolshed.g2.bx.psu.edu/repository/browse_repository?id=66db1d2dd5ec1fab) |    Copernicus Climate Change Service (C3S)   |  

```

Comment:
some sentences are finishing in this table with `.` and some others not

Reply:
removed the dots.

Reply:
Thanks